apps/condo/lang/ru/ru.json

```diff
@@ -1314,12 +1314,15 @@
   "incident.form.prompt.exit.message": "Запись не сохранится, если вы уйдете со страницы",
   "incident.form.prompt.exit.title": "Не сохранять запись?",
   "incident.form.save.label": "Сохранить",
+  "incident.generateNews.switch.hint": "В новой вкладке откроется черновик новости, текст которой сгенерирован на основе информации об этом отключении",
```

Comment:
```suggestion
  "incident.generateNews.switch.hint": "В новой вкладке откроется черновик новости, сгенерированной на основе этого отключения",
```

Reply:
For now, I think I'll leave the Russian translations as they are.